client/reader/tags/trending-tags.tsx

```diff
@@ -1,7 +1,8 @@
+import formatNumber from '@automattic/components/src/number-formatters/lib/format-number';
 import { addLocaleToPathLocaleInFront } from '@automattic/i18n-utils';
+import { getLocaleSlug } from 'i18n-calypso';
```

Comment:
There are a few issues with using `getLocaleSlug()` directly for getting the locale slug:

* It's shared between the client and the server so it might cause unexpected issues with translations on the server (and trending tags is SSR-ed as part of the `/tags` page)
* `getLocaleSlug()` is just a method so if the locale changes, it won't trigger a re-render, which might cause obsolete strings to be displayed.

with that in mind, I'd recommend either:

* Using the `useTranslate()` hook for functional components, it also returns a `localeSlug` property.
* Using the `localize()` high-order component for class components, it also returns a `localeSlug` property.

Hat tip to @yuliyan for the extra info on this.

Reply:
Ah, never knew that! I've made those changes now - let me know if I've misunderstood what you're getting at. :)

Reply:
Thanks - that change seems alright. The only concern I have is around the inconsistent use of the locale slug - sometimes we'll pass it to `formatNumber()`, sometimes we won't, and that may lead to inconsistent results when a non-EN locale is currently used for the user's WP.com interface.